components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/builder/WSO2LoginEventPayloadBuilder.java

```diff
@@ -17,53 +17,207 @@
 import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.organization.management.service.exception.OrganizationManagementException;
 import org.wso2.identity.webhook.common.event.handler.builder.LoginEventPayloadBuilder;
 import org.wso2.identity.webhook.common.event.handler.model.EventData;
+import org.wso2.identity.webhook.common.event.handler.util.EventHookHandlerUtils;
+import org.wso2.identity.webhook.wso2.event.handler.constant.Constants;
+import org.wso2.identity.webhook.wso2.event.handler.internal.WSO2EventHookHandlerDataHolder;
+import org.wso2.identity.webhook.wso2.event.handler.model.AuthenticationFailedReason;
 import org.wso2.identity.webhook.wso2.event.handler.model.WSO2AuthenticationFailedEventPayload;
 import org.wso2.identity.webhook.wso2.event.handler.model.WSO2AuthenticationSuccessEventPayload;
+import org.wso2.identity.webhook.wso2.event.handler.model.common.Application;
+import org.wso2.identity.webhook.wso2.event.handler.model.common.Organization;
+import org.wso2.identity.webhook.wso2.event.handler.model.common.User;
+import org.wso2.identity.webhook.wso2.event.handler.model.common.UserClaim;
+import org.wso2.identity.webhook.wso2.event.handler.model.common.UserStore;
+import org.wso2.carbon.identity.application.authentication.framework.context.AuthenticationContext;
+import org.wso2.carbon.identity.application.authentication.framework.model.AuthenticatedUser;
+import org.wso2.carbon.identity.application.common.model.ClaimMapping;
+import org.wso2.carbon.identity.core.util.IdentityTenantUtil;
 import org.wso2.identity.event.common.publisher.model.EventPayload;
 
 import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
+import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.ErrorMessages.ERROR_CODE_INVALID_ORGANIZATION_ID;
+import static org.wso2.carbon.identity.role.v2.mgt.core.RoleConstants.Error.UNEXPECTED_SERVER_ERROR;
 import static org.wso2.identity.webhook.common.event.handler.constant.Constants.EVENT_SCHEMA_TYPE_WSO2;
 
 /**
  * WSO2 Login Event Payload Builder.
  */
 public class WSO2LoginEventPayloadBuilder implements LoginEventPayloadBuilder {
 
+    private static final Log log = LogFactory.getLog(WSO2LoginEventPayloadBuilder.class);
+
     @Override
     public EventPayload buildAuthenticationSuccessEvent(EventData eventData) throws IdentityEventException {
 
-        //TODO: Implement the logic to build the authentication success event payload.
+        AuthenticationContext authenticationContext = eventData.getAuthenticationContext();
+        AuthenticatedUser authenticatedUser = eventData.getAuthenticatedUser();
+
+        if (authenticatedUser == null) {
+            throw new IdentityEventException("Authenticated user cannot be null.");
+        }
+
+        User user = new User();
+        try {
+            populateUserAttributes(authenticatedUser, user);
```

Comment:
Take this out of this block.
If user id is not present attributes will also not present in this case

Reply:
Addressed with https://github.com/Sachin-Mamoru/identity-webhook-event-handlers/pull/1/commits/8750eff9ee3a84ff76fd11671bbbc3451fbb0fc8